### PR TITLE
Set theme selector max-height

### LINKED ISSUE #2441: Scroll in theme panel
In GMF 2.1.1, is the theme panel scrollable ?

If not, I think it would be good to set a max size and add a scroll bar if this size is reached.

### PR DESCRIPTION
**Pull request by fredj**
fixes #2441

### CODE REVIEW

File: contribs/gmf/less/desktop.less [hunk 1]
```diff
@@ -211,6 +211,9 @@ gmf-search {
 
 gmf-themeselector {
   width: 1.5 * @left-panel-width;
+  max-height: 1.5 * @left-panel-width;
```
Comment (ger-benjamin) [inline contribs/gmf/less/desktop.less]: @left-panel-width ? Is that correcton a small height screen ?

Comment (fredj) [inline contribs/gmf/less/desktop.less]: it's just to have (at max) a square div with a scrollbar. this size is arbitrary (I've checked with @ybolognini )

Comment (ger-benjamin) [inline contribs/gmf/less/desktop.less]: OK

**Comment by kalbermattenm** [comment]
Thanks a lot @fredj

**Pull request merged**